charts/admission-controller/templates/scanner/psp.yaml

```diff
@@ -2,7 +2,8 @@
 apiVersion: policy/v1beta1
```

Comment:
Don't hate me but could you please add the k8s version check to this PSP policy? Like what we did for the agent https://github.com/sysdiglabs/charts/blob/master/charts/agent/templates/psp.yaml#L1
Otherwise the deploy on Kubernetes v1.25 will fail due to the deprecation of PSP policies

Reply:
done!